platform/website/src/routes/about/+page.svelte

```diff
@@ -0,0 +1,340 @@
+<script lang="ts">
+	import BlogPost from "$/components/about/blog-post.svelte";
+	import Footer from "$/components/about/footer.svelte";
+	import HeroSection from "$/components/about/hero-section.svelte";
+	import LogoText from "$/components/icons/logo-text.svelte";
+	import { PUBLIC_BLOG_API_KEY } from "$env/static/public";
+	import { faGithub } from "@fortawesome/free-brands-svg-icons";
+	import {
+		faCircleExclamation,
+		faCode,
+		faHandHoldingDollar,
+		faHeart,
+		faPenNib,
+		faVideo,
+	} from "@fortawesome/free-solid-svg-icons";
+	import Fa from "svelte-fa";
+
+	// https://ghost.org/docs/content-api
+	const BLOG_ENDPOINT = `https://bytes.scuffle.tv/ghost/api/content/posts/?key=${PUBLIC_BLOG_API_KEY}&include=authors&limit=2&fields=title,primary_author,url,excerpt,published_at`;
+
+	async function fetchPosts() {
+		const res = await fetch(BLOG_ENDPOINT, {
+			headers: {
+				Accept: "application/json",
+			},
+		});
+		return await res.json();
+	}
+</script>
+
+<svelte:head>
+	<title>Scuffle</title>
+</svelte:head>
+
+<div class="content">
+	<header>
+		<a class="logo" href="/">
+			<LogoText />
+		</a>
+	</header>
+	<main>
+		<HeroSection />
+		<div class="features">
+			<section class="emotes">
+				<div class="caption">
+					<span>EMOTES</span>
+					<Fa icon={faHeart} />
+				</div>
+				<h2>Emotes for everyone</h2>
+				<span>
+					Effortless emotes for all users out of the box. No need to setup any complicated
+					third-party apps.
+				</span>
+			</section>
+			<section class="community">
+				<div class="caption">
+					<span>COMMUNITY FIRST</span>
+					<Fa icon={faVideo} />
+				</div>
+				<h2>By viewers, for viewers</h2>
+				<span>
+					Built by the community, for the community. We're always listening to feedback and
+					suggestions.
+				</span>
+			</section>
+			<section class="blog">
+				<div class="heading">
+					<h2>Scuffle Engineering Blog</h2>
+					<Fa icon={faPenNib} />
+				</div>
+				<div class="posts">
+					{#await fetchPosts()}
+						<BlogPost />
+						<BlogPost />
+					{:then posts}
+						{#each posts.posts as post}
+							<BlogPost data={post} />
+						{/each}
+					{:catch error}
+						<div class="error">
+							<Fa icon={faCircleExclamation} />
+							Failed to fetch posts: {error}
+						</div>
+					{/await}
+				</div>
+				<a class="read-more" href="https://bytes.scuffle.tv/">
+					Read more
+					<span class="arrow">-></span>
+				</a>
+			</section>
+			<section class="code">
+				<div class="caption">
+					<span>OPEN SOURCE</span>
+					<Fa icon={faCode} />
+				</div>
+				<h2>Contributions welcome</h2>
+				<span>Scuffle is open source, meaning anyone can contribute to the project.</span>
```

Comment:
is there a better way to say this kinda don't want it to sound off putting by having to explain it